src/Laravel/WebHookArtisanCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Telegram\Bot\Laravel;
+
+use Illuminate\Console\GeneratorCommand as LaravelGeneratorCommand;
+use Symfony\Component\Console\Input\InputOption;
+use Telegram\Bot\Api;
+use Telegram\Bot\BotsManager;
+
+class WebHookArtisanCommand extends LaravelGeneratorCommand
```

Comment:
Please rename this to `WebhookCommand` (Notice **Webhook** Not **WebHook**). Obviously update everywhere.

Perhaps, we could also move this into its own namespace like `Telegram\Bot\Laravel\Artisan` as I think we can add some more useful commands in near future. Thanks!